### PR TITLE
feat(encoder): Show many addresses at once

### PR DESCRIPTION
**Pull request by ukint-vs**
Rendered: https://ss58.org/pr-31/ (need to upload pretty-checkbox css)

- Show many addresses for predefined networks
- Use checkbox for custom prefix

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <link data-trunk rel="copy-dir" href="webfonts/" />
     <link rel="stylesheet" href="/css/bulma.min.css"/>
     <link rel="stylesheet" href="/css/fas.min.css">
+    <link rel="stylesheet" href="/css/pretty-checkbox.min.css"/>
```
Comment (shamilsan) [inline index.html:14]: Do we really need it? As for me Bulma checkbox is ok too: https://bulma.io/documentation/form/checkbox/

File: src/encoder.rs [hunk 2]
```diff
@@ -98,66 +120,47 @@ impl Component for Encoder {
...
-                        <span class="icon"><i class="fas fa-user"></i></span>
-                        <span>{ "Alice" }</span>
-                    </button>
-                    <button class="button is-danger" onclick={ ctx.link().callback(|_| Msg::Clear) }>
-                        <span class="icon"><i class="fas fa-times"></i></span>
-                        <span>{ "Clear" }</span>
-                    </button>
+                </span>
+                <div class="control p-1">
+                    <input class="input is-info is-small" type="number" min="0" max="16383" disabled={ !self.custom_prefix }
```
Comment (shamilsan) [inline src/encoder.rs:142]: As for me, regular input looks better
```suggestion
                    <input class="input is-info" type="number" min="0" max="16383" disabled={ !self.custom_prefix }
```

**Review by shamilsan — APPROVED** [review]
Awesome 🔥 

I have a minor rendering problem in Firefox with checkbox text wrapping: 
image

**Comment by ukint-vs** [comment]
> Awesome 🔥
> 
> I have a minor rendering problem in Firefox with checkbox text wrapping: image

image

It will look like this with css

File: src/encoder.rs (hunk 2 above)
Comment (ukint-vs) [reply src/encoder.rs:142]: image

I think the regular size is a little distracting

**Comment by shamilsan** [comment]
Ah, I got it; I looked without the new CSS.

File: index.html (hunk 1 above)
Comment (ukint-vs) [reply index.html:14]: Idk. It's pretty but 18kb size

Comment (ukint-vs) [reply index.html:14]: Maybe there is a way to leave only used styles for my switch checkbox. I'll look into it.

Comment (shamilsan) [reply index.html:14]: No, that's not a problem. My problem was that I looked at https://ss58.org/pr-31/, but the CSS file hasn't been merged yet; therefore, I didn't see the difference. It was my mistake, sorry.

**Pull request merged**